registry/handlers/app.go

```diff
@@ -441,6 +441,7 @@ func (app *App) register(routeName string, dispatch dispatchFunc) {
 // configureEvents prepares the event sink for action.
 func (app *App) configureEvents(configuration *configuration.Configuration) {
 	// Configure all of the endpoint sinks.
+	// nolint:prealloc
```

Comment:
Can we add a comment explaining the nolint?

Reply:
It's similar to the earlier describe `nolint` case above: if the endpoint is disabled we `continue` with evaluation the next one so we do not know the  exact size at the time the iteration starts - we can, once again, preallocate to the capacity of `configuration.Notifications.Endpoints` and make peace with a bit of extra memory potentially not being used. Happy to change.

Reply:
Added a comment: https://github.com/distribution/distribution/pull/4037/commits/823cd4a370e2aa7fe9d4bccc0cae9940bb88fa84